code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -352,6 +362,26 @@
 	proc/on_finish(mob/eater)
 		return
 
+	proc/drop_item(var/path)
+		var/obj/drop = new path
+		if(istype(drop))
+			drop.pixel_x = src.pixel_x
+			drop.pixel_y = src.pixel_y
+			var/obj/item/I = drop
+			if(istype(I))
+				var/mob/M = src.loc
+				if(istype(M))
+					var/item_slot = M.get_slot_from_item(src)
+					if(item_slot)
+						M.u_equip(src)
+						src.loc = null
+						var/mob/living/carbon/human/H = M
+						if(istype(H))
```

Comment:
best practice for this is to use ishuman() and if so, cast

Reply:
i'll change this bit to use ishuman() because we already established src.loc isn't null with the earlier check